script/Ciel/Day 13/13_01_02_CIEL13_6.txt

```diff
@@ -650,52 +782,59 @@ When I open my eyes, I see that the Dead are already reaching toward me.
 [sha:c3a36dc702431dc3112ca323b06c75da8cc281d6]{
 -- Page 153, Offset 35684.
 -- 地面に倒れこんだ俺に殺到する手、手、手。
-Their hands swarm upon me as I lay collapsed on the ground.
+From where I lie on the ground,
+ I can see their groping, grasping hands pour down like rain.
 }
 [sha:ff8e5750a0d10b3afcc61f42570b6c31a78a5a79]{
 -- Page 153, Offset 35685.
 -- 倒れたまま転がって鉄柵から離れる。
-I roll away from the iron fence without getting back on my feet.
+Without getting up, I roll away from the iron fence.
 }
 [sha:8f0c8936f7bb4dd8cd874f04ac8c669a5ad4e6d6]{
 -- Page 153, Offset 35686.
 -- ノエルの指示なのか、死者たちは鉄柵の中には入ってこない。
-The Dead aren't going past the fence, likely because Noel ordered them not to.
+Noel must've ordered the Dead to stay put, as they aren't crossing over the fence.
 }
 [sha:99a7f8708b2344e14c54634d66014c050f0ee0ed]{
 -- Page 153, Offset 35687.
 -- となれば、間髪入れずにノエル自身の追撃が来る―――！
-Which means that Noel's going to come after me without a moment's notice―――!
+Which means... Noel herself is probably hot on my tail―――!
 }
 [sha:2495980b660ff3a88e76ad1769a66f4173c6ec6b]{
 -- Page 154, Offset 35688.
 -- 「あ―――ああああ―――ああもう、最っっっ高！」
-"AH―――AAAAH―――Ah, how incredible!"
+"%{i}AH%{/i}――%{i}AAAAH%{/i}――%{i}Ah%{/i}, I can't get enough!"
 }
 [sha:a47b003b9d8ed82826a18b498d147ee28df3d4b2]{
 -- Page 154, Offset 35689.
 -- ……追撃は来なかった。
-...She isn't coming after me.
+She isn't coming after me...
 }
 [sha:c7a95fb6ffd403e87f635b7ea8505255be6889d5]{
 -- Page 154, Offset 35690.
 -- 円陣の中心には、自分の体を抱きしめる吸血鬼がひとり。
-The lone vampire stands in the middle of the circle, embracing her own body tightly.
+The vampire stands alone in the dead center of the circle,
+ hugging her own body tightly.
 }
 [sha:3d07616fb3a9bf652ce2db21596cb753b166d2c8]{
 -- Page 155, Offset 35691.
 -- 「これが吸血鬼の体なのね！　イメージ通りに体が動いて、イメージ以上の力に満ちている……！
-"So this is a body of a vampire! My body is moving exactly as I envision it, and it's brimming with more energy than I can even imagine...!
+"So this is the body of a vampire!
+ I imagined it would be able to move like this,
+ but I never could have imagined this %{i}power%{/i}...!
 }
 [sha:c5c467ecaceaaf16a0a4dddc5cb32b6187ee8112]{
 -- Page 155, Offset 35692.
 -- わたし、ずっと疑問だった！　どうして吸血鬼たちはあんなに偉そうで、ドイツもコイツも人間を虫ケラ扱いするのかって！　でも、その理由が分かっちゃった！」
-It's been bothering me the entire time! I always wondered why vampires are so proud of themselves, and why they all treat humans like they're insects! But now, I finally understand why they feel that way!"
+It had been bothering me this entire time!
+ I just didn't understand why vampires always acted so high and mighty,
+ and why they treated humans like they're insects!
+ But now, I finally get it!"
```

Comment:
Think it could be simplified quite a bit.
>I %{i}always%{i} wondered!
>Why do vampires act sooo high and mighty——why do all of them treat humans like insects?!